### PR TITLE
THE NEW YORK TIMES ARTICLE PAGE

### PR DESCRIPTION
**Pull request by smy5152**
**WEEK 1:
Goals:**
### Version Control, HTML & CSS Overview and playground.

Dear Reviewer, 
Kind request to provide feedback on our collaborative project. Appreciate the extra-effort and support from the Microverse Team.

It was an amazing experience & the more time I spent with the code I found myself addicted to not leaving it incomplete.

I was not at all confident about the linters, that was a first-time experience and I successfully managed to clear all errors.

Skills Revived & Learned:
- Versioning & best practices of maintaining branches of safe deployed and local dev
- git flow & Github flow
- HTML, CSS new and better elements, semantics, features etc
- Cloning - the hard work and skills needed to successfully make a clone.
- Linter - install & fix tests _{still not managed to get styleint to run on VSC locally}_
- Collaborative learning, being patient, helping each other succeed.
- Attending Daily standup, being focused, accountable.

Thanks 
Micronaut Team 
Shabbir & Chukwuma

### CODE REVIEW

File: assets/styles.css [hunk 1]
```diff
@@ -1,3 +1,409 @@
-.header {
- color: red;
-}
\ No newline at end of file
+body {
+  margin: 0;
+  /* font-family: ""; */
```
Comment (arslanbisharat) [inline assets/styles.css]: - [x] Commented code is not a good practice in clean coding. Please avoid it.

File: assets/styles.css [hunk 2]
```diff
@@ -1,3 +1,409 @@
...
+  display: flex;
+  vertical-align: baseline;
+}
+
+.header-left {
+  float: left;
+  display: flex;
+  flex: 0.2;
+  justify-content: space-evenly;
+  /* align-items: center; */
```
Comment (arslanbisharat) [inline assets/styles.css]: - [x]  Commented code is not a good practice in clean coding. Please avoid it.

File: assets/styles.css [hunk 3]
```diff
@@ -1,3 +1,409 @@
...
+  width: max-content;
+  float: left;
+  display: block;
+  cursor: pointer;
+}
+
+.header-right {
+  float: left;
+  display: flex;
+  /* padding: 5px; */
```
Comment (arslanbisharat) [inline assets/styles.css]: Commented code is not a good practice in clean coding. Please avoid it.

File: assets/styles.css [hunk 4]
```diff
@@ -1,3 +1,409 @@
...
+  margin-right: 5px;
+  width: max-content;
+  flex: 0.2;
+  justify-content: space-evenly;
+  /* align-items: center; */
```
Comment (arslanbisharat) [inline assets/styles.css]: - [x]  Commented code is not a good practice in clean coding. Please avoid it.

File: assets/styles.css [hunk 5]
```diff
@@ -1,3 +1,409 @@
...
+  font-weight: 700;
+  font-size: 1.2rem;
+  width: calc(100% - 40px);
+  margin: 0 20px 20px;
+  max-width: 600px;
+}
+.socialToolBar {
+  border-top: 1px solid #ebebeb;
+  display: flex;
+  /* flex-direction: row; */
```
Comment (arslanbisharat) [inline assets/styles.css]: - [x]  Commented code is not a good practice in clean coding. Please avoid it.

File: assets/styles.css [hunk 6]
```diff
@@ -1,3 +1,409 @@
...
+  width: 600px;
+  margin: 0 20px 20px;
+  /* align-items: center; */
+  /* max-width: 600px; */
```
Comment (arslanbisharat) [inline assets/styles.css]: - [x]  Commented code is not a good practice in clean coding. Please avoid it.

File: assets/styles.css [hunk 7]
```diff
@@ -1,3 +1,409 @@
...
+}
+.socialToolBar a {
+  line-height: 65px;
+  padding: 0 5px;
+  /* color: white; */
```
Comment (arslanbisharat) [inline assets/styles.css]: - [x]  Commented code is not a good practice in clean coding. Please avoid it.

File: assets/styles.css [hunk 8]
```diff
@@ -1,3 +1,409 @@
...
+  display: inline;
+  float: left;
+  /* transition: all 0.5s; */
```
Comment (arslanbisharat) [inline assets/styles.css]: - [x]  Commented code is not a good practice in clean coding. Please avoid it.

File: assets/styles.css [hunk 9]
```diff
@@ -1,3 +1,409 @@
...
+  align-items: center;
+  cursor: pointer;
+}
+.socialIcons {
+  /* margin-left: 350px; */
```
Comment (arslanbisharat) [inline assets/styles.css]: - [x]  Commented code is not a good practice in clean coding. Please avoid it.

File: assets/styles.css [hunk 10]
```diff
@@ -1,3 +1,409 @@
...
+  align-self: center;
+  vertical-align: middle;
+  padding: 5px;
+  width: 15px;
+  height: 15px;
+  /* width: 15px; */
+  /* height: 15px; */
+  /* max-width: fit-content; */
+  /* justify-self: center;
+  text-align: center; */
```
Comment (arslanbisharat) [inline assets/styles.css]: - [x]  Commented code is not a good practice in clean coding. Please avoid it.

File: assets/styles.css [hunk 11]
```diff
@@ -1,3 +1,409 @@
...
+  text-transform: capitalize;
+  width: 150px;
+  font-size: 14px;
+  font-weight: 500;
+  height: 23px;
+  line-height: 16px;
+}
+.subscribe {
+  border-left: 1px solid rgb(226, 226, 226);
+  /* flex: 1 1 0%; */
```
Comment (arslanbisharat) [inline assets/styles.css]: - [x]  Commented code is not a good practice in clean coding. Please avoid it.

File: assets/styles.css [hunk 12]
```diff
@@ -1,3 +1,409 @@
...
+  margin: 2px 0px 0px;
+  padding: 10px 0px 0px;
+  border-top: 1px solid rgb(226, 226, 226);
+}
+.end-article-block {
+  width: 600px;
+  margin: 0 auto;
+}
+.end-article-text {
+  /* font-family: nyt-franklin, helvetica, arial, sans-serif; */
```
Comment (arslanbisharat) [inline assets/styles.css]: - [x]  Commented code is not a good practice in clean coding. Please avoid it.

File: assets/styles.css [hunk 13]
```diff
@@ -1,3 +1,409 @@
...
+  font-size: 0.75rem;
+  /* line-height: 1rem; */
+  width: calc(100% - 60px);
+  max-width: 600px;
+  /* margin: 0px auto 1em; */
```
Comment (arslanbisharat) [inline assets/styles.css]: - [x]  Commented code is not a good practice in clean coding. Please avoid it.

File: assets/styles.css [hunk 14]
```diff
@@ -1,3 +1,409 @@
...
+  border-top: 1px solid rgb(226, 226, 226);
+}
+.end-article-block {
+  width: 600px;
+  margin: 0 auto;
+}
+.end-article-text {
+  /* font-family: nyt-franklin, helvetica, arial, sans-serif; */
+  font-size: 0.75rem;
+  /* line-height: 1rem; */
```
Comment (arslanbisharat) [inline assets/styles.css]: - [ ]  Commented code is not a good practice in clean coding. Please avoid it.

File: assets/styles.css [hunk 15]
```diff
@@ -1,3 +1,409 @@
...
+  line-height: 65px;
+  padding: 0 5px;
+  /* color: white; */
+  display: inline;
+  float: left;
+  /* transition: all 0.5s; */
+  align-items: center;
+  cursor: pointer;
+}
+.socialIcons {
```
Comment (arslanbisharat) [inline assets/styles.css]: - [x] This rule is empty. Avoid it for clean coding purposes.

File: assets/styles.css [hunk 16]
```diff
@@ -1,3 +1,409 @@
...
+  border-radius: 100%;
+  margin-top: 10px;
+  width: 15px;
+  height: 15px;
+}
+.bookmark_comment a i::before {
+  border-color: rgb(102, 102, 102) transparent;
+  border-style: solid;
+  bottom: -7px;
+  border-width: 6px 0px 0px 8px;
```
Comment (arslanbisharat) [inline assets/styles.css]: - [x] Values of 0 shouldn't have units specified. Please remove px. with 0.

File: assets/styles.css [hunk 17]
```diff
@@ -1,3 +1,409 @@
...
+  content: "";
+  display: block;
+  left: 15px;
+  position: absolute;
+  width: 0px;
```
Comment (arslanbisharat) [inline assets/styles.css]: - [x] Values of 0 shouldn't have units specified. Please remove px. with 0.

File: assets/styles.css [hunk 18]
```diff
@@ -1,3 +1,409 @@
...
+}
+.bookmark_comment a i {
+  height: 15px;
+  margin-top: 12px;
+}
+.bookmark_comment a i::after {
+  border-color: rgb(255, 255, 255) transparent;
+  border-style: solid;
+  border-width: 6px 0px 0px 6px;
```
Comment (arslanbisharat) [inline assets/styles.css]: - [x] Values of 0 shouldn't have units specified. Please remove px. with 0.

File: assets/styles.css [hunk 19]
```diff
@@ -1,3 +1,409 @@
...
+  bottom: -6px;
+  content: "";
+  display: block;
+  left: 16px;
+  position: absolute;
+  width: 0px;
```
Comment (arslanbisharat) [inline assets/styles.css]: - [x] Values of 0 shouldn't have units specified. Please remove px. with 0.

File: assets/styles.css [hunk 20]
```diff
@@ -1,3 +1,409 @@
...
+  content: "";
+  display: block;
+}
+.nav-header {
+  display: flex;
+  flex-flow: row;
+  height: 45px;
+}
+.go-to-home {
+  margin: 18px 0px 0px auto;
```
Comment (arslanbisharat) [inline assets/styles.css]: - [x] Values of 0 shouldn't have units specified. Please remove px. with 0.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Request changes 😳

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_

This is a great opportunity to learn. Microverse is here to help you. 

**Please have a look at the suggestions to get more understanding.**
- [x]  Values of 0 shouldn't have units specified. Please remove px. with 0. 
- [x] Please make your Readme Descriptive. You have used a great template btw but you forgot to edit it accordingly. Please work on it. 
- [x] You have linter errors. Please remove them before asking for any code review request. 
- [x] Please don't use commented code. This is not a good practice of writing clean code. 
- [x] In your clone, you have missed many sections. Please work on it and add missing sections. 
- [x] Please enlarge the button present in the navbar. 
- [x] 1.	Add more descriptive information to your PR message. Please tell me something interesting about the TSEs and what you liked about the project. 
2.	The bullet points are nice to add.




**Please make the changes and request a re-review for the given project**

If you get stuck somewhere, don't hesitate to reach me out or schedule s 1:1 meeting with a Full-time TSE. 

Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

File: assets/styles.css (hunk 1 above)
Comment (smy5152) [reply assets/styles.css]: Done.

File: assets/styles.css (hunk 3 above)
Comment (smy5152) [reply assets/styles.css]: Done.

File: assets/styles.css (hunk 5 above)
Comment (smy5152) [reply assets/styles.css]: Done.

File: assets/styles.css (hunk 7 above)
Comment (smy5152) [reply assets/styles.css]: Done.

File: assets/styles.css (hunk 6 above)
Comment (smy5152) [reply assets/styles.css]: Done.

File: assets/styles.css (hunk 8 above)
Comment (smy5152) [reply assets/styles.css]: Done.

File: assets/styles.css (hunk 15 above)
Comment (smy5152) [reply assets/styles.css]: Done.

File: assets/styles.css (hunk 9 above)
Comment (smy5152) [reply assets/styles.css]: Done.

File: assets/styles.css (hunk 10 above)
Comment (smy5152) [reply assets/styles.css]: Done.

File: assets/styles.css (hunk 16 above)
Comment (smy5152) [reply assets/styles.css]: Done.

File: assets/styles.css (hunk 17 above)
Comment (smy5152) [reply assets/styles.css]: Done.

File: assets/styles.css (hunk 18 above)
Comment (smy5152) [reply assets/styles.css]: Done.

File: assets/styles.css (hunk 19 above)
Comment (smy5152) [reply assets/styles.css]: Done.

File: assets/styles.css (hunk 20 above)
Comment (smy5152) [reply assets/styles.css]: Done.

File: assets/styles.css (hunk 11 above)
Comment (smy5152) [reply assets/styles.css]: Done.

File: assets/styles.css (hunk 12 above)
Comment (smy5152) [reply assets/styles.css]: Done.

File: assets/styles.css (hunk 13 above)
Comment (smy5152) [reply assets/styles.css]: Done.

**Review by smy5152 — COMMENTED** [review]
All comments and requests to fix errors and update missing content completed.

File: assets/styles.css [hunk 21]
```diff
@@ -1,3 +1,767 @@
...
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  font-weight: 700;
+  font-size: 1.0625rem;
+  line-height: 1.3125rem;
+  padding-top: 20px;
+  border-bottom: 1px solid rgb(226, 226, 226);
+}
+
+.suggestions {
+  /* width: 80%; */
```
Comment (andresporras3423) [inline assets/styles.css]: Remove commented line of code

File: assets/styles.css [hunk 22]
```diff
@@ -1,3 +1,767 @@
...
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  font-weight: 700;
+  font-size: 1rem;
+  line-height: 1.25rem;
+}
+
+.css-y3qb10 {
+  margin: 0;
+}
+
```
Comment (andresporras3423) [inline assets/styles.css]: Group this selector and selector on line 1, both have the same style definition

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Smymisr Yamani & Chukwuma Obasi** 
Good job with the improvements for this project :muscle:  :muscle:
Your page meets most of the requirements :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:

- [x] Remove commented code in the CSS file
- [x] Remove horizontal scrollbar



- [x] Group selectors with the same style definition
- [x] Your project should align elements with a combination of all three techniques (float, flex and grid).
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

File: assets/styles.css (hunk 21 above)
Comment (smy5152) [reply assets/styles.css]: Done

File: assets/styles.css (hunk 22 above)
Comment (smy5152) [reply assets/styles.css]: Done

**Review by umairarshadbutt — APPROVED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
### The [clone](https://smy5152.github.io/microverse/) shows a good understanding and usage of Embedding images, box model, Positioning elements with float, flex and grid. :100:
##
- The cloned page is rendering as it should and meets all the project specifications.
- Pull Request has a meaningful title and a short description.
- CSS is clean. No linter errors at this point and no bad practices used as well.
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @smy5152  and @Chubie ! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
**MIT license in the README file has 404 Error, please fix it before merge**
##
**If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by smy5152** [comment]
That was an amazing experience.

Thanks a lot and appreciate the feedback & code reviews.

Regards
Micronauts: @smy5152 and @Chubie

**Pull request merged**